src/o365/spo/commands/site/site-classic-add.ts

```diff
@@ -427,6 +427,8 @@ class SpoSiteClassicAddCommand extends SpoCommand {
     log(
       `  ${chalk.yellow('Important:')} to use this command you have to have permissions to access
     the tenant admin site.
+
+    This command is deprecated. Please use ${chalk.yellow('spo site add')} instead.
```

Comment:
To make it even clearer that a particular command is deprecated, we show a warning when the deprecated command is executed. See https://github.com/pnp/office365-cli/blob/8a47cfef5aca71fbd3a920729bc63554c69963d0/src/o365/cli/commands/cli-consent.ts#L41 for an example

Reply:
Tried to add this command but all tests fail afterwards. Do you have a suggestion how to fix these tests?

Reply:
It would be interesting to investigate why the tests were failing. Do you have any more information about the errors you got when running tests after this addition?

Reply:
No, it seems that the vorpal calls assume that something has been called if I am adding the line above before all other http calls. 

Not sure how to tell him to ignore the first call in the vorpal checks

Reply:
Fixed this, by adding the deprecation warning the checks for some tests were no more correct. I was able to fix it now